### ISSUE TITLE
A11y_.NETCore_RecipesAppMyRecipesNew_NewRecipe_ScreenReader: Talkback is not announcing "Recipe Name" field as required field and not marked up with any chevron/symbol.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#15625]
### Description

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_.NETCore_Android_May23; #.NET Core; #WCAG1.3.1; #Android; #AndroidPhone; #Talkback;

### Environment Details:​
Application Name: Rachel's Recipe
Operating System: Android 13
Screen Reader: Talkback

### Repro Steps:

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till "New" button and double tap.
4. "New Recipe" Screen will open.
5. Start Talkback.
6. Swipe till "Recipe Name" edit field and observe the issue.

### Actual Results:
Talkback is not announcing "Recipe Name" field as required field and not marked up with any chevron/symbol.

### Expected Results:
Talkback should announce "Recipe Name" field as required field and it should be marked up with any chevron/symbol.

### User Impact:
Screen reader users will not get information about the required field and will not understand whether it is required field.

### Attachment:
[Video 6-19 at 12.40 (1).webm](https://github.com/dotnet/maui/assets/90901519/e0549379-600c-4547-872e-c1a92e7e5f1b)

### Link to public reproduction project repository

Test

### Version with bug

Unknown/Other

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 13

**Comment by PoojaNamde** [comment]
Verified the bug in latest environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment.

#Closed; #Regressed:07-25-23;
